lib/vdsm/v2v.py

```diff
@@ -970,8 +970,9 @@ def _parse_progress(self, chunk):
         m = self.DISK_PROGRESS_RE.match(chunk)
         if m is None:
             return None
+        value = [x for x in m.groups() if x is not None][0]
```

Comment:
Is there any risk of this failing because the regexp wouldn't match? Previously, it was part of the following try block, now maybe it could fail with IndexError rather than OutputParserError?

Reply:
No, that's a misunderstanding. If the regex does not match it is  caught by the `if m is None` statement above and not the `try` block below. This is still true after the change. Also, the way the regex is defined there is no change that the regex would match but all the groups were None -- exactly one is always not None.

Reply:
Ah, right, sorry.